bootstrap/pgbouncer/main.tf

```diff
@@ -18,12 +18,30 @@ variable "dbsync_probe_image_tag" {
   default = "27a9dbc30253e7d2036f1d6648d406f3d17a90e2"
 }
 
-variable "pg_bouncer_replicas" {
-  default = 1
+variable "dns_zone" {
+  default = "demeter.run"
+}
+
+variable "extension_name" {
+  default = "dbsync-v3"
+}
+
+variable "instance_role" {
+  default = "pgbouncer"
+}
+
+variable "network" {
+  type    = string
+  default = "preview"
+}
```

Comment:
Also just to keep in mind, I think this PR doesn't include the change to have a pgbouncer per network, so I think this module shouldn't receive the `network` variable